test/SlowTests/Issues/RavenDB-19101.cs

```diff
@@ -0,0 +1,151 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using FastTests;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Http;
+using Raven.Client.ServerWide.Operations.Certificates;
+using Sparrow.Json;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+using HttpMethod = System.Net.Http.HttpMethod;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_19101: RavenTestBase
+{
+    public RavenDB_19101(ITestOutputHelper output) : base(output)
+    {
+    }
+    
+    private class GetCertificateFromIndexCommand : RavenCommand<object>
```

Comment:
Why is it called `GetCertificate...`? Wouldn't `GetIndexHistoryCommand` be more suitable?

Reply:
Done